templates/aspnet-s2i-template.json

```diff
@@ -280,7 +277,7 @@
         {
             "name": "ASP_UPSTREAM_IMAGE_TAG",
             "description": "Base OpenShift ASP S2I Image Tag",
-            "value": "latest",
+            "value": "1.0",
```

Comment:
if the template itself is defining the imagestream, then i tend to agree, it doesn't matter.